README.rst

```diff
@@ -272,6 +272,18 @@ Configuration variables can be set via Env variables:
 - ``WATCHER_SCALYR_JOURNALD_EXTRA_FIELDS``: Add extra Systemd Journald fields. Should be a JSON string. Example: '{"_COMM": "command"}'
 - ``WATCHER_SCALYR_JOURNALD_PATH``: Journald logs path mounted from the host. (Default: ``/var/log/journald``)
 
+Scalyr custom parser
+....................
+
+In some cases you might need to assign a **custom** Scalyr parser for specific container. This can be achieved via Pod annotations. The following example shows an annotation value that instructs kubernetes-log-watcher to set custom parser ``json-java-parser`` for container ``app-1``.
+
+.. code-block:: yaml
+
+  annotations:
+    kubernetes-log-watcher/scalyr-parser: '[{"container": "app-1", "parser": "json-java-parser"}]'
```

Comment:
I think an annotation name like: `log-watcher.kubernetes.io/scalyr-parser` is more in line with how people do it everywhere else, though I'm not completely sure if `kubernetes.io` is the right namespace. But I think it's better than for instance `zalando.org` for this component.